src/plugins/messenger/MessengerPreview/components/MessengerImage/components/ImagePreview.tsx

```diff
@@ -0,0 +1,64 @@
+import { useMessangerImageContext } from '../MessangerImageContext';
+import { MessagePluginFactoryProps } from "../../../../../../common/interfaces/message-plugin";
+import { getBackgroundImage } from '../../../lib/css';
+
+export const getImagePreview = ({ React, styled }: MessagePluginFactoryProps) => {
+
+    const templateStyle = ({ template = 'media' }) => ({
+        media: {
+            paddingTop: "75%"
+        },
+        list: {
+            paddingTop: "50%"
+        },
+        generic: {
+            paddingTop: "50%",
+            display: "table",
+            width: "100%"
+        }
+    })[template];
+
+    const FixedImage = styled.div({
+        backgroundSize: "cover",
+        backgroundPosition: "center center",
+        cursor: 'pointer'
+    }, templateStyle)
+
+    const FlexImage = styled.img({
+        display: 'block',
+        width: '100%',
+        cursor: 'pointer'
+    })
+
+    const ImagePreview = () => {
+        const { config, url, altText, template, onExpand } = useMessangerImageContext();
+
+        const handleKeyDown = (event: KeyboardEvent) => {
+            event.code === "Enter" && onExpand && onExpand();
+        }
+
+        return config.settings.dynamicImageAspectRatio ? (
+                <FlexImage
```

Comment:
We can improve the focus style of these images and make it consistent with the rest of the elements. We can remove the outline and add an inset box shadow that uses the primary weak color from the theme (For example. 'box-shadow: inset 0 0 0px 1px <theme.primaryWeakColor>'. Similar to focusing audio and video elements)

Reply:
right now I just added some opacity on hover and focus. Can be improved later on next commit